### ISSUE TITLE
Problem in formatISO documentation

### ISSUE THREAD

**Issue by reintroducing** [date-fns/date-fns#2965]
There is an issue with the [formatISO documentation](https://date-fns.org/v2.28.0/docs/formatISO)

  ### The problem with formatISO documentation

The documentation states this:
```js
// Represent 18 September 2019 in ISO 8601 format, time only (local time zone is UTC):
const result = formatISO(new Date(2019, 8, 18, 19, 0, 52), { representation: 'time' })
//=> '19:00:52Z'
```
Executing that very code in the console comes back with `'19:00:52-07:00'` and not `'19:00:52Z'`.

  ### Suggested edits

I'm not sure if the method is outputting incorrectly and there is a bug in the codebase or if the documentation is wrong. I am, however, trying to figure out how to get the format that the documentation states should be the output and am not sure how to achieve that.

**Comment by mandaputtra** [comment]
`Z` will only return if the timezone offset from `new Date().getTimezoneOffset()`   are returned `0 (zero)`

**Comment by reintroducing** [comment]
Yes, without knowing that info, I was expecting it to return what it stated and that was not the case.

**Comment by RobSchilderr** [comment]
[Related](https://github.com/date-fns/date-fns/issues/2151) to this?
